io/net/bonding.py

```diff
@@ -112,6 +112,8 @@ def setUp(self):
         self.peer_wait_time = self.params.get("peer_wait_time", default=5)
         self.sleep_time = int(self.params.get("sleep_time", default=5))
         self.mtu = self.params.get("mtu", default=1500)
+        self.physical_interface = self.params.get("physical_interface",
+                                                  default=True)
```

Comment:
@vaishnavibhat Sorry for looking into this very late.

Can you use a sysfs or ethtool command logic to decide whether an interface is physical or virtual please ?